image.php

```diff
@@ -40,12 +40,10 @@
 
 			wp_link_pages(
 				array(
-					'before'      => '<div class="page-links"><span class="page-links-title">' . esc_html__( 'Pages:', 'twentytwentyone' ) . '</span>',
-					'after'       => '</div>',
-					'link_before' => '<span>',
-					'link_after'  => '</span>',
-					'pagelink'    => '<span class="screen-reader-text">' . esc_html__( 'Page', 'twentytwentyone' ) . ' </span>%',
-					'separator'   => '<span class="screen-reader-text">, </span>',
+					'before'   => '<nav class="page-links" aria-label="' . esc_attr__( 'Page', 'twentytwentyone' ) . '">',
+					'after'    => '</nav>',
+					/* translators: There is a space after page. */
+					'pagelink' => esc_html__( 'Page ', 'twentytwentyone' ) . '%',
```

Comment:
I think the % should be in each of the translation strings, so that the number & text can be controlled by the translator (in case there's a language where "3 Page" makes more sense)

Reply:
> I think the % should be in each of the translation strings, so that the number & text can be controlled by the translator (in case there's a language where "3 Page" makes more sense)

+1, makes sense. Good catch!
Should be in a new PR since this one was already merged. Apologies, I missed your comment by a few seconds.